Dockerfile.assisted-installer

```diff
@@ -1,8 +1,16 @@
-FROM quay.io/centos/centos:centos8
-copy build/installer /
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
+ENV GOFLAGS=-mod=mod
+WORKDIR /go/src/github.com/openshift/assisted-installer
+COPY . .
+RUN make installer
+
+FROM registry.svc.ci.openshift.org/ocp/4.7:base
 ARG GIT_REVISION
-LABEL "git_revision"=${GIT_REVISION}
 ARG QUAY_TAG_EXPIRATION
+
+COPY --from=builder /go/src/github.com/openshift/assisted-installer/build/installer /usr/bin/installer
+COPY --from=builder /go/src/github.com/openshift/assisted-installer/deploy/assisted-installer-controller /assisted-installer-controller
```

Comment:
I think that the controller isn't needed, it is part of the 2nd image. no?

Reply:
these `${PROJECT_ROOT}/deploy/assisted-installer-controller` are the deployment manifests for the controller right?

Reply:
You are probably right. From the name, I thought it's an executable.